xgo-scripts/latest.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+FILES=$(find `pwd`/build/binary/* -type f)
+echo $BUILDID
+
+for currentfile in ${FILES}; do
+	latestfile=${currentfile/$BUILDID/latest}
```

Comment:
Nitpick, but it looks like the indentation in this block is a mix of tabs and spaces.